responsibleai/responsibleai/rai_insights/rai_insights.py

```diff
@@ -324,6 +332,16 @@ def _validate_model_analysis_input_parameters(
                 "Expecting pandas Dataframe for train and test."
             )
 
+    def _validate_features_same(self, small_train_features_before,
+                                small_train_data, function):
```

Comment:
1. Can this function take the first two parameter as list of features? Looks a weird function signature with one being list and other being dataframe.

2. Could you add a function description and parameters descriptions and types?

Reply:
1.) I could, but I would have the same duplicate code twice though, so I think this is preferred as it reduces code duplication?
2.) done!  This is an internal function though and I see other internal functions which don't have docs, but probably good to add it anyway.